### PR TITLE
Adding a link to the developer docs

### PR DESCRIPTION
**Pull request by cwattrus**
Adding a link to the Developer Docs from the navigation links.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -211,6 +214,7 @@ <h2>open_api_parser</h2>
                     <h4>
                     <span class="item__lang">Ruby</span>
                     <span class="item__lang">Shell</span>
+                    <span class="item__lang">Dockerfile</span>
```
Comment (crookedneighbor) [inline index.html]: Dockerfile? or just Docker?

File: index.html [hunk 2]
```diff
@@ -298,12 +302,12 @@ <h1>Join the team.</h1>
             </nav>
 
             <small class="legal">Braintree is a service of PayPal, Inc. © 2008 - <span id="current-year"></span> PayPal, Inc.
-        <a target="_blank" href="https://www.paypal.com/uk/webapps/mpp/ua/privacy-full">Privacy Policy</a> | <a target="_blank" href="https://www.braintreepayments.com/legal">Legal</a></small>
+        <a target="_blank" href="https://www.braintreepayments.com/legal/braintree-privacy-policy">Privacy Policy</a> | <a target="_blank" href="https://www.braintreepayments.com/legal">Legal</a></small>
```
Comment (cwattrus) [inline index.html]: @crookedneighbor when you changed the legal stuff with Zac. Did you do it in the handlebars file that generates this? I'm wondering why this code has changed. This index.html is generated from the index handlebars file in source. The only new thing here should be the link above.

File: index.html (hunk 1 above)
Comment (cwattrus) [reply index.html]: @crookedneighbor This is generated by the handlebars file below. I am not super familiar with the code, but I believe this is pulled from GitHub.

File: index.html (hunk 2 above)
Comment (crookedneighbor) [reply index.html]: We probably didn't realize that it was generated via handlebars.

**Comment by cwattrus** [comment]
@crookedneighbor no idea. It was not updated by me.

**Pull request merged**